Automattic/wp-calypso
# Desktop: update IPC usage to Electron v0.35

In Electron v0.35 `require( 'ipc' )` is deprecated, and the new way is to include electron directly.

To test:
- Build Calypso and ensure no errors are thrown regarding the `require( 'electron' )` (it is defined as an external in the Makefile)
- Build the desktop app with this branch and ensure that it works
